presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeProbabilityFunctionQueries.java

```diff
@@ -87,7 +87,7 @@ public void testInverseBetaCDF()
     public void testNormalCDF()
     {
         assertQuery("SELECT normal_cdf(nationKey, 7.3, 10.5) FROM nation");
-        assertQuery("SELECT normal_cdf(9.15, nationKey, 5.3) FROM nation");
+        assertQuery("SELECT normal_cdf(9.15, nationKey, 5.3) FROM nation WHERE nationKey > 0");
```

Comment:
It might be better to add a predicate nationKey != 0 for this. But even if this throws an error, does that not match the java side ?

Reply:
@aditi-pandit I wondered this too. Shouldn't this test pass as long as the thrown errors match?

Presto ([MathFunctions.java](https://github.com/prestodb/presto/blob/833ecd0c62d3fdcd4be90620edd2d18dbda41eb9/presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java#L819))
```
    public static double normalCdf(
            @SqlType(StandardTypes.DOUBLE) double mean,
            @SqlType(StandardTypes.DOUBLE) double standardDeviation,
            @SqlType(StandardTypes.DOUBLE) double value)
    {
        checkCondition(standardDeviation > 0, INVALID_FUNCTION_ARGUMENT, "standardDeviation must be > 0");
        return 0.5 * (1 + Erf.erf((value - mean) / (standardDeviation * Math.sqrt(2))));
    }
```

Velox ([Probability.h](https://github.com/facebookincubator/velox/blob/39ec3b02d5c58028ee1391555f77e766cdfc49ed/velox/functions/prestosql/Probability.h#L64))
```
  ...
  call(double& result, double m, double sd, double value) {
    VELOX_USER_CHECK_GT(sd, 0, "standardDeviation must be > 0");

    static const double kSqrtOfTwo = sqrt(2);
    result = 0.5 * (1 + erf((value - m) / (sd * kSqrtOfTwo)));
  }
  ...
```

Reply:
@rrando901 : `assertQuery` method does not check for matching error messages. It only checks the result rows match. The alternative would be to call `assertQueryFails` with the failure message. 

The current changes are fine for now.

Reply:
Added the suggested predicate `nationKey != 0`